src/components/Entry.vue

```diff
@@ -231,7 +327,10 @@ export default {
 
     /** Whether the current typed ID is valid */
     isInvalidLoginMessageShown() {
-      return !this.isCurrentUserValid && this.validateCount == 1;
+      return (
+        (!this.isCurrentUserValid && this.validateCount == 1) ||
+        (!this.isCurrentUserValid && this.validateCount == 0)
+      );
```

Comment:
why is this being done?

Reply:
Like discussed in call, added a hack check in the `signup` PR.